### DISCUSSION TITLE
Welcome to DBeaver Discussions!

### DISCUSSION THREAD

**Discussion by serge-rider** [dbeaver/dbeaver#10637] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  We used to use issues with label `question` for discussions.
  Now we will use this new GitHub feature.

  Feel free to create new questions here!

**Comment by MarlboroB** [comment]
Hi. Starting with DBeaver version 7.2.2, routines are not shown in mysql databases. In General, only the list of tables is expanded, and the rest of the folders (routines. indexes and so on) are empty. I checked on different windows and linux computers. Does anyone have a solution to this problem?

**Reply by serge-rider** [reply]
Please create a bug report in issues. Generally I can't reproduce this issue, more details is required (including driver and server versions)
